helix-term/src/commands/typed.rs

```diff
@@ -1824,6 +1824,111 @@ fn run_shell_command(
     Ok(())
 }
 
+fn help(cx: &mut compositor::Context, args: &[Cow<str>], event: PromptEvent) -> anyhow::Result<()> {
+    if event != PromptEvent::Validate {
+        return Ok(());
+    }
+
+    if args.is_empty() {
+        // TODO: Open a list of commands?
+        todo!()
+    }
```

Comment:
For now this could be an `ensure!(!args.is_empty(), "Select a help topic")`

It would be nice to have an overview page for :help in the long-run that gives some introductory info about helix ("here's how to use the tutor / command palette / :help command / etc")

Reply:
I went ahead and wrote up a help file for the `:help` command itself and made it just open that if there are no arguments.